### PR TITLE
Add bright black color

### LINKED ISSUE #48: blackBright doesn't exist
ref chalk/chalk#257

### PR DESCRIPTION
**Pull request by nmschulte**
with gray as alias
apply grey/gray human fix for background

--

not sure how to add attribution

Fixes #48

### CODE REVIEW

**Comment by nmschulte** [comment]
for #48 and chalk/chalk#257

not sure the code-style issue with `xo` but I addressed the issues w/in `ansi-styles`

**Comment by nmschulte** [comment]
@Qix- -- poke!

**Comment by Qix-** [comment]
Ah yeah no problem, node 4 is no longer in LTS so xo got upgraded to use newer syntax.

File: index.js [hunk 1]
```diff
@@ -39,9 +39,11 @@ function assembleStyles() {
 			magenta: [35, 39],
 			cyan: [36, 39],
 			white: [37, 39],
-			gray: [90, 39],
 
 			// Bright color
+			blackBright: [90, 39],
+			// Alias bright black as gray
+			gray: [90, 39],
 			redBright: [91, 39],
```
Comment (Qix-) [inline index.js]: Please move the alias to below the bright section.

File: index.js [hunk 2]
```diff
@@ -62,6 +64,8 @@ function assembleStyles() {
 
 			// Bright color
 			bgBlackBright: [100, 49],
+			// Alias bright black as gray
+			bgGray: [100, 49],
```
Comment (Qix-) [inline index.js]: Same thing here

File: index.js [hunk 3]
```diff
@@ -72,8 +72,9 @@ function assembleStyles() {
 		}
 	};
 
-	// Fix humans
-	styles.color.grey = styles.color.gray;
+	// Alias bright black as gray (and grey)
+	styles.color.grey = styles.color.gray = styles.color.brightBlack;
+	styles.bgColor.bgGrey = styles.bgColor.bgGray = styles.bgColor.bgBrightBlack;
```
Comment (Qix-) [inline index.js]: Code style forbids chained assignment - could you break them out onto their own lines?

Comment (nmschulte) [reply index.js]: updated

**Comment by sindresorhus** [comment]
@nmschulte Can you fix the merge conflict and document the new properties in the readme?

**Comment by nmschulte** [comment]
> @nmschulte Can you fix the merge conflict and document the new properties in the readme?

@sindresorhus done and done.

**Comment by sindresorhus** [comment]
Sorry for the inconvenience, but this will have to wait for https://github.com/chalk/chalk/issues/300.

**Pull request merged**